src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -465,6 +470,41 @@ def _get_failed_targets_from_junitxml(self, junitxml, targets):
 
     return failed_targets
 
+  def _get_test_info_from_junitxml(self, junitxml):
+    tests_in_path = []
+
+    SUCCESS = 0
```

Comment:
These magic values should be strings instead, and should probably move up to `TestRunnerTaskMixin`.
```
SUCCESS = 'success'
FAILURE = 'failure'
ERROR = 'error'
```
The loss in readability of magic values in the JSON outweighs any possible efficiency gains of using integers. If need be, internal systems can map from string constants to custom values.